packages/python-packages/apiview-stub-generator/README.md

```diff
@@ -16,14 +16,20 @@ Following are the steps to create an API review request for python package.
 ### Generate stub File
 `apiview-stub-generator` package is used to generate stub file tokens from either source code repo or from prebuilt wheel package. Following are the steps to generate stub file token.
 
-Install `apiview-stub-generator` package. From the root of the apiview-stub-generator package, run:
+Install `apiview-stub-generator` package on a supported Python version. If not run on a supported Python version, you may see "Running apiview-stub-generator version ...", even though the tool is not actually running.
+
+From the root of the apiview-stub-generator package, run:
+```
+(venv310) apiview-stub-generator> pip install . --extra-index-url="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/"
```

Comment:
`(venv310) apiview-stub-generator>`

Cut this part since it's not what someone would actually type.